### ISSUE TITLE
Default image

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/unraid-templates#2]
I'm not sure I had already asked, but how did you build the image used here @aronwk-aaron  ? I think this was the regular image with the configs read from the env variables, right? (in that case we can use the regular ones as that was merged into master many moons ago)

**Comment by aronwk-aaron** [comment]
Yeah, I just made it where it will try to pickup environmental variables that are passed into the docker container
